lib/CustomFields/components/FieldAccordion/edit-fields/shared-fields/HiddenField.js

```diff
@@ -17,10 +17,11 @@ const HiddenField = ({ fieldNamePrefix }) => (
   <Col xs={3}>
     <Field
       type="checkbox"
-      component={Checkbox}
       name={`${fieldNamePrefix}.hidden`}
       label={<FormattedMessage id="stripes-smart-components.customFields.hidden" />}
-    />
+    >
+      {props => <Checkbox {...props} vertical />}
+    </Field>
```

Comment:
Why you tossing child component in this way? It is correctly works with `vertical` prop in Field component
```
<Field
  type="checkbox"
  component={Checkbox}
  name={`${fieldNamePrefix}.hidden`}
  label={<FormattedMessage id="stripes-smart-components.customFields.hidden" />}
  vertical
/>
```
Some modules could has warning on rest props, this way can omit warnings in future